base/shell/explorer/explorer.cpp

```diff
@@ -87,6 +87,45 @@ HideMinimizedWindows(IN BOOL bHide)
 }
 #endif
 
+static BOOL
+IsExplorerSystemShell()
+{
+    HKEY hKeyWinlogon;
+	WCHAR szShell[256];
+	WCHAR szPath[MAX_PATH];
+    LPWSTR szExplorer = NULL;
```

Comment:
```suggestion
    LPWSTR szExplorer;
```
it's going to be unconditionally initialized by the PathFindFileName call from below.